docs/sources/reference/components/prometheus/prometheus.exporter.windows.md

```diff
@@ -198,21 +192,21 @@ The v2 collector can query service states much more efficiently, but can't provi
 
 ### smb block
 
-Name           | Type     | Description                                                    | Default | Required
----------------|----------|----------------------------------------------------------------|---------|---------
-`enabled_list` | `string` | Deprecated (no-op), Comma-separated list of collectors to use. | `""`    | no
+Name           | Type           | Description                  | Default | Required
+---------------|----------------|------------------------------|---------|---------
+`enabled_list` | `list(string)` | A list of collectors to use. | `[]`    | no
```

Comment:
The collect is not deprecated. And there are no plan to deprecate it. Only the option `enabled_list` where removed, because there was only one sub-collector which make the whole flag pointless.

Reply:
Thank you, I brought back the "deprecated" note for `enabled_list` in the `smb` block.